rocAL/rocAL/source/parameters/parameter_factory.cpp

```diff
@@ -116,6 +116,13 @@ ParameterFactory::get_seed()
     return _seed;
 }
 
+void
+ParameterFactory::regenerate_seed()
+{
```

Comment:
Parenthesis style needs to be changed as per google coding guidelines

Reply:
The entire file follows a different standard, hence leaving this as is for now

Reply:
The previous API has paranthesis near the function def, so please check